launch/launch/exit_handler.py

```diff
@@ -35,6 +35,10 @@ def default_exit_handler(context, ignore_returncode=False):
     if not context.launch_state.teardown:
         context.launch_state.teardown = True
 
+    __default_exit_handler_helper(context)
```

Comment:
The keyword argument `ignore_returncode` must be passed to the new internal function:

```
__default_exit_handler_helper(context, ignore_returncode=ignore_returncode)
```